crates/pipeline_manager/src/runner.rs

```diff
@@ -125,6 +132,9 @@ impl Display for RunnerError {
             Self::PipelineStartupError { pipeline_id, error } => {
                 write!(f, "Failed to start pipeline '{pipeline_id}': '{error}'")
             }
+            Self::PipelineShutdownError { pipeline_id, error } => {
```

Comment:
Let's remove single quotes around `{error}` everywhere.  They are not helpful and look bad.